### ISSUE TITLE
Vitis fails to open

### ISSUE THREAD

**Issue by RossComputerGuy** [flathub/com.github.corna.Vivado#2]
Whenever I click "Tools" -> "Launch Vitis IDE", I notice an error message is logged while running `flatpak run com.github.corna.Vivado`. This is the error.
```
/app/share/xilinx/xilinx_helpers.sh: line 166: zenity: command not found
```
Please add zenity to make Vitis work.

**Comment by corna** [comment]
Are you sure you are running the latest version? It works for me, and zenity has been added in f90252ff8685d58ddd38cd106d8248aa1ccc025a.

**Comment by RossComputerGuy** [comment]
```
[ross@lavienrose:~]$ flatpak info com.github.corna.Vivado

Xilinx Vivado Design Suite - Xilinx Design Suite for hardware development

          ID: com.github.corna.Vivado
         Ref: app/com.github.corna.Vivado/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 1.0.0
     License: LicenseRef-proprietary
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 15.9 MB
     Runtime: org.gnome.Sdk/x86_64/42
         Sdk: org.gnome.Sdk/x86_64/42

      Commit: 04de7e7c55b9567d55e77ee4f4e12a69020876a98af1d786089ab8dff99c76a7
      Parent: 4a8e5da335a783e270bc44e85d0807b07c761324c02c765f079163b439efb7a3
     Subject: Replace "username" with "e-mail" in the login dialogs and errors (9f027d9e)
        Date: 2022-05-04 15:53:48 +0000
```

I ran `flatpak update` and Vivado isn't listed as having any updates needed.

**Comment by RossComputerGuy** [comment]
I have figured out that Zenity is in `/app/bin` and not `/usr/bin`.

**Comment by corna** [comment]
You are right, I forgot to update that line when moving to Gnome 42 as runtime. Fixed in c0ea9830aa09985389907a2ce501e4ff7c121c38.

Note that `zenity` is invoked there only if a particular tool is not installed, so you likely did not select vitis during the installation of the Xilinx tools.
